wire-ios-transport/Source/PushChannel/StarscreamPushChannel.swift

```diff
@@ -80,6 +80,8 @@ class StarscreamPushChannel: NSObject, PushChannelType {
     }
 
     func reachabilityDidChange(_ reachability: ReachabilityProvider) {
+        WireLogger.backend.debug("reachability did change. May be reachable: \(reachability.mayBeReachable), is mobile connection: \(reachability.isMobileConnection)")
```

Comment:
Can we add one more log into `ReachabilityWrapper` to make sure we don't have no real reachability?

Reply:
done